src/schema/rules/sidecars/eyetrack.yaml

```diff
@@ -0,0 +1,64 @@
+#
+# Groups of related metadata fields
+#
+# Assumptions: never need disjunction of selectors
+# Assumptions: top-to-bottom overrides is sufficient logic
+
+---
+EyeTrackingRecommended:
+  selectors:
+    - modality == "eyetrack"
+    - datatype == "eyetrack"
+    - suffix == "eyetrack"
+  fields:
+    Manufacturer:
+      level: recommended
+      description: |
+        The sampling frequency of data channels that deviate from the main
+         sampling frequency SHOULD be specified in the `channels.tsv` file.
```

Comment:
There is no channels.tsv file though?

Reply:
this was a mistake of copy paste, corrected, thanks to spot it.